info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2023-01-28T00:33:41.814Z"}
info: Connected to the database. {"timestamp":"2023-01-28T00:33:44.276Z"}
error: 
    status - 500 
    message - Error: Error: Invalid Credentials
    at _callee3$ (C:\Users\RAHUL KUMAR\projjjject\awesome-project\src\services\/user.service.js:45:11)
    at tryCatch (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-28T00:35:34.515Z"}
error: 
    status - 500 
    message - Error: Error: Invalid Credentials
    at _callee3$ (C:\Users\RAHUL KUMAR\projjjject\awesome-project\src\services\/user.service.js:45:11)
    at tryCatch (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-28T00:36:37.119Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2023-01-28T01:09:55.990Z"}
info: Connected to the database. {"timestamp":"2023-01-28T01:09:58.452Z"}
error: 
    status - 500 
    message - Error: ReferenceError: bcrypt is not defined
    at _callee3$ (C:\Users\RAHUL KUMAR\projjjject\awesome-project\src\services\/user.service.js:45:11)
    at tryCatch (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-28T01:10:45.658Z"}
error: 
    status - 500 
    message - Error: ReferenceError: bcrypt is not defined
    at _callee3$ (C:\Users\RAHUL KUMAR\projjjject\awesome-project\src\services\/user.service.js:45:11)
    at tryCatch (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:44:17)
    at Generator.<anonymous> (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:125:22)
    at Generator.next (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:69:21)
    at asyncGeneratorStep (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\RAHUL KUMAR\projjjject\awesome-project\node_modules\@babel\runtime\helpers\asyncToGenerator.js:22:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2023-01-28T01:11:04.215Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2023-01-28T01:11:41.594Z"}
info: Connected to the database. {"timestamp":"2023-01-28T01:11:43.965Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2023-01-28T01:15:48.707Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2023-01-28T01:16:03.484Z"}
info: Connected to the database. {"timestamp":"2023-01-28T01:16:05.680Z"}
info: Server started at http://localhost:3000/api/v1/ {"timestamp":"2023-01-28T02:57:28.445Z"}
info: Connected to the database. {"timestamp":"2023-01-28T02:57:30.691Z"}
